source/PlatformTracker.hx

```diff
@@ -0,0 +1,36 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.input.keyboard.FlxKey;
+
+/**
+ * ...
+ * @author Jess Geiger
+ */
+
+ class PlatformTracker extends Player
```

Comment:
OPTIONAL FIX: 

I don't think this PlatformTracker class really needs to extend Player.
It doesn't use any of player's variables or methods in any situation. It doesn't need a hitbox or a graphic either, which are things that come with inheriting from Player (which inherited from FlxSprite). 
This really is just a little bundle of data, so I don't think you need it to inherit from anything.